client/landing/stepper/declarative-flow/trial-wooexpress-flow.ts

```diff
@@ -66,33 +73,6 @@ const wooexpress: Flow = {
 			} catch {}
 		}
 
-		const getLoginUrl = () => {
-			const redirectTo = addQueryArgs(
-				`${ window.location.protocol }//${ window.location.host }/setup/wooexpress`,
-				{
-					...Object.fromEntries( queryParams ),
-				}
-			);
-
-			let logInUrl = login( {
```

Comment:
I am not sure if we can switch this flow over as-is, because it is currently redirecting users to `/log-in` rather than the `/start/account/user[-social]` flow.

Reply:
Points I found about it:

1) The flow works when we point the user to /account/user instead of the login.
2) If we pass the same attributes to the sign-in page (e.g. `wccom-from` ) and the user selects "login" all attributes are passed to the login page.
3) After the login or sign the flow works properly.
4) Search about the params usage, I found they are very related to the old relation between woocomerce.com and wp.com login. [#88608](https://github.com/Automattic/wp-calypso/pull/88608)  and [#woocommerce.com/19925](https://github.com/Automattic/woocommerce.com/pull/19925) .
This flow was deprecated recently, so we don't have more relation between both sites.


Considering the factors above we can:
1) Keep the current change,  since the params are passed between the sign-in and login page.
2) Adapt the stepper to make it possible to select the login instead of the sign-in page.

I'm more inclined towards option 2.
What do you think?

Reply:
@daledupreez Any thoughts about it?

Reply:
Apologies for the delay!

I would prefer to vote for 1, as it would keep things simpler. However, I think we need a more flexible approach like option 2, as we have multiple flows that specify alternate login URLs, including `wooexpress`, `videopress`, and `videopress-tv`. So I think we need something more flexible for this.

Reply:
@daledupreez 
> we have multiple flows that specify alternate login URLs, including wooexpress, videopress, and videopress-tv. So I think we need something more flexible for this.

https://github.com/Automattic/wp-calypso/pull/92390 to fix the videopress flow.

As soon as the PR #92390 is merged I can update this flow again.

Reply:
@daledupreez  PR url with the option 2 approach. Enabling the usage of a custom login path